client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -4,31 +4,29 @@
  */
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
 import { includes } from 'lodash';
-import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import Banner from 'components/banner';
-import FoldableCard from 'components/foldable-card';
-import Card from 'components/card';
 import CompactCard from 'components/card/compact';
-import SectionHeader from 'components/section-header';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import ExternalLink from 'components/external-link';
+import FoldableCard from 'components/foldable-card';
 import FormFieldset from 'components/forms/form-fieldset';
 import FormLegend from 'components/forms/form-legend';
-import CompactFormToggle from 'components/forms/form-toggle/compact';
 import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import InfoPopover from 'components/info-popover';
-import ExternalLink from 'components/external-link';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
 import QueryJetpackConnection from 'components/data/query-jetpack-connection';
 import QuerySiteRoles from 'components/data/query-site-roles';
-import { getStatsPathForTab } from 'lib/route';
+import SectionHeader from 'components/section-header';
+import { activateModule } from 'state/jetpack/modules/actions';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 import { getSiteRoles } from 'state/site-roles/selectors';
-import { activateModule } from 'state/jetpack/modules/actions';
+import { getStatsPathForTab } from 'lib/route';
 import {
 	isActivatingJetpackModule,
 	isJetpackModuleActive,
```

Comment:
Do we want to hide the `View your site stats` link if the module is inactive or unavailable? cc @Automattic/jetpack-design

Reply:
@tyxla I'd keep the link — it's still useful for discovery and if there were any stats collected before it's a major reason to re-enable stats. In case it's inactive, we could add a notice near the stats itself letting the user know.

What do you mean by “unavailable” though? Are there certain scenarios where I can't turn on stats?

Reply:
Modules can be disabled in code by a developer, or by a host. We are working to make sure our UI responds to those conditions.

Reply:
I'd still keep it unless we can tell:

- We haven't collected any stats before for this site.
- We can tell the stats module is “unavailable” rather than anything else rendering it “inactive”.